formula_renames.json

```diff
@@ -129,5 +129,6 @@
   "v8-315": "v8@3.15",
   "wires": "geckodriver",
   "wpcli-completion": "wp-cli-completion",
-  "xu4": "scummvm"
+  "xu4": "scummvm",
+  "ht-rust": "xh"
```

Comment:
The rest of the list is sorted alphabetically; can we do the same with `ht-rust`? Thanks.

Reply:
Fixed